### PR TITLE
Slash and slide vincent

### PR DESCRIPTION
**Pull request by vincentliok**
Left click to do upward slash

### CODE REVIEW

**Comment by vincentliok** [comment]
Followed design doc

File: Assets/Scripts/SlashAndSlide.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+        // Start slash animation
+    }
+
+    public void SlashContact(GameObject other)
+    {
+        sliding = true;
+        if (other.TryGetComponent<Slashable>(out Slashable slashable))
+        {
+            dashDir = slashable.dashDir;
```
Comment (TheStress) [inline Assets/Scripts/SlashAndSlide.cs]: Seems incomplete? like there is no end to the sliding

File: Assets/Scripts/Sword.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Sword : MonoBehaviour
+{
+    public SlashAndSlide player;
+
+    private void OnCollisionEnter(Collision collision)
```
Comment (TheStress) [inline Assets/Scripts/Sword.cs]: I check it in Unity and the collider you are using is a trigger so use that instead

File: Assets/Scripts/Slashable.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Slashable : MonoBehaviour
```
Comment (TheStress) [inline Assets/Scripts/Slashable.cs]: Nothing to change but keep in mind that there might be conflicts with the `Slashable` Benicio made

**Pull request merged**